MMM-MBTA.js

```diff
@@ -124,7 +128,7 @@ Module.register("MMM-MBTA", {
             row.appendChild(symbolCell);
 
             var descCell = document.createElement("td");
-            descCell.innerHTML = "No vehicles are scheduled";
+            descCell.innerHTML = "Nothing coming soon";
```

Comment:
Missing a period or a "..."

```suggestion
            descCell.innerHTML = "Nothing coming soon...";
```